model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceModel.scala

```diff
@@ -87,7 +87,7 @@ case class WorkspaceRequest (
 case class Workspace(
                       namespace: String,
                       name: String,
-                      authorizationDomain: Option[ManagedGroupRef],
+                      authorizationDomain: Set[ManagedGroupRef],
```

Comment:
there was a discussion in the project team slack room a little while ago about having the authorizationDomain be one thing that can contain between 0 and n groups. see https://broadinstitute.slack.com/archives/G3EJQNF17/p1499264371830532